recipes/asl/meta.yaml

```diff
@@ -0,0 +1,40 @@
+{% set name = "asl" %}
```

Comment:
anyone who's written a solver for ampl perhaps. lots of people use ampl without being aware of the implementation details at all.